commit 86217fbc58e9bfb23025cafd876fcc0bb9759c2e (refs/changes/54/4654/1)
Author: Cody Peter Mello <cody.mello@joyent.com>
Date:   2018-08-08T18:15:06+00:00 (1 year, 2 months ago)
    
    TRITON-671 SAPI should fall back to local store when Moray returns NoDatabasePeersError

diff --git a/lib/server/stor/moray_local.js b/lib/server/stor/moray_local.js
index 6618dde..81fc5f8 100644
--- a/lib/server/stor/moray_local.js
+++ b/lib/server/stor/moray_local.js
@@ -49,6 +49,7 @@ function is500(err) {
     //               ^^^^^ <-- Is what should be in the list below
     var names = [
         'NoConnectionError',     // Moray HA Proxy is down
+        'NoDatabasePeersError',  // Moray has no backend connections
         'ConnectionClosedError', // Moray services are down
         'PoolClosedError',       // Postgres is down
         'Error',         // Postgres is down
